lineman/app/models/group_records_interface.coffee

```diff
@@ -6,3 +6,6 @@ angular.module('loomioApp').factory 'GroupRecordsInterface', (BaseRecordsInterfa
       @fetch
         path: "#{parentGroup.id}/subgroups"
         cacheKey: "subgroupsFor#{parentGroup.key}"
+
+    uploadPhoto: (group, file, kind) =>
+      @remote.upload("#{group.key}/upload_photo/#{kind}", file).then @remote.onSuccess, @remote.onFailure
```

Comment:
Also I think this should be on GroupModel.. since it is a method that works on a single group.. then there is one less argument to it also.

I'm hoping we'll be able to remove all the recordsInterfaces soon.. I'll come up with a nice way to put all the various fetch methods in a block on the model